### ISSUE TITLE
Make coincident edges curved

### ISSUE THREAD

**Issue by vincerubinetti** [greenelab/hetmech-frontend#47]
It seems to be very rare, but sometimes the same two nodes can have more than one edge connecting them. In this case, we might consider making the lines curvy to avoid them overlapping.

I consider this to be a somewhat low priority issue, just because there is a much more common and problematic issue of edges overlapping when there are many nodes in the graph, which is a problem that is very difficult or unsolvable.

@dhimmel if you know of any node pairs that have a lot of coincident edges, that'd be helpful for testing.

**Comment by dhimmel** [comment]
There are three possible relationship types between Compounds and Genes as well as Diseases to Genes. Genes to Genes also have multiple relationship types.

Try this cypher query to find Compound-Gene pairs with multiple relationships:

```
MATCH p=(source:Compound)-[rel]->(target:Gene)
RETURN
  source.name,
  target.name,
  count(rel) AS n_edges
ORDER BY n_edges DESC
LIMIT 100
```

Those nodes could be good testing candidates.

**Comment by vincerubinetti** [comment]
Here is a demo:
https://jsfiddle.net/1braL9ug/

**Comment by dhimmel** [comment]
Nice! The behavior looks good and versatile!

**Comment by vincerubinetti** [comment]
closed by #50
